common/djangoapps/student/views.py

```diff
@@ -2479,6 +2491,7 @@ def reactivation_email_for_user(user):
     subject = ''.join(subject.splitlines())
     message = render_to_string('emails/activation_email.txt', context)
     from_address = configuration_helpers.get_value('email_from_address', settings.DEFAULT_FROM_EMAIL)
+    from_address = configuration_helpers.get_value('activation_email_from_address', from_address)
```

Comment:
One more thing, we should remember to set this value as `'edX <activate@edx.org>'` so that the sender appears as "edX" in the recipient's email client.